### ISSUE TITLE
Timer trigger: Only once and immediate case doesn't work

### ISSUE THREAD

**Issue by torresashjian** [TIBCOSoftware/flogo-contrib#60]
I am getting the following error  When scheduling a Only once and immediate timer:

Error scheduleOnce flo err:  cannot set recurrent time with 0

**Comment by torresashjian** [comment]
I don't know if anybody has made any changes to it. We can close it as duplicate of #170 if you want to have just one bucket! @mellistibco

**Comment by mellistibco** [comment]
nah, more interested in resolving problems vs duplicate gh issues. i'll test it now

**Comment by mellistibco** [comment]
closing, cant seem to reproduce the exact error
